.github/workflows/release-process.yml

```diff
@@ -39,11 +39,20 @@ jobs:
           ./hack/build-release.sh
           cd cli
           ./hack/build-binaries.sh
+
+      - name: Run Package build
+        run: |
+          constraintVersion="${{ github.ref_name }}"
+          ./cli/kctrl-linux-amd64 pkg release -y -v ${constraintVersion:1} --debug
+          cd cli
           shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
           echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
           echo '```' | tee -a ../tmp/checksums-formatted.txt
           cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
           echo '```' | tee -a ../tmp/checksums-formatted.txt
+          cd ..
+          shasum -a 256 carvel-artifacts/packages/kapp-controller.carvel.dev/package.yml | tee -a tmp/checksums.txt
+          shasum -a 256 carvel-artifacts/packages/kapp-controller.carvel.dev/metadata.yml | tee -a tmp/checksums.txt
```

Comment:
We should probably add these 2 to the checksums-formatted.txt? (which will add them to the release body)

Reply:
nit: Also, should we use `./carvel-artifacts/...` to be consistent?